content/events-reference/events-reference-v5/product-models-50.md

```diff
@@ -2,7 +2,122 @@
 TODO
 
 ## Product model created
-TODO
+
+### Format
+A product model created event follows this format:
+::: event_api_reference content/events-reference/events-reference-serenity/resources/product-model-created.yml
```

Comment:
```suggestion
::: event_api_reference content/events-reference/events-reference-serenity/resources/product-model-created-50.yml
```

Reply:
We should have 2 YML files because the product-model-created.yml will evolve throughout 2021. On the contrary, the product-model-created-50.yml won't evolve.

Reply:
I forgot to do the same comment on the product.update PR.